content/en/apps/reference/api.md

```diff
@@ -1045,6 +1045,7 @@ curl http://localhost:5988/api/v1/monitoring
 | `outbound_push.backlog` | Number | Number of changes yet to be processed by Outbound Push. |
 | `feedback.count` | Number | Number of feedback docs created usually indicative of client side errors. |
 | `conflict.count` | Number | Number of doc conflicts which need to be resolved manually. |
+| `replication_limit.count` | Number | Number of users that exceeded the replication limit of documents. |
```

Comment:
This is kind of confusing to me. From what I can tell, the `count` that it returns is the number of docs replicated by user, _not_ the number of users that exceeded the limit. Am i missing something?